chrome/content/zotero/xpcom/http.js

```diff
@@ -311,6 +311,47 @@ Zotero.HTTP = new function() {
 				if (options.debug) {
 					Zotero.debug(xmlhttp.responseText);
 				}
+				
+				// Try to use HTTP redirect, but if url extraction fails
+				// or numRedirects is exceeded, just return the current page
+				if (options.responseType === 'document' &&
+					(!options.numRedirects || options.numRedirects < 3)) {
+					let contentType = xmlhttp.getResponseHeader('Content-Type');
+					if (contentType && contentType.startsWith('text/html')) {
+						let meta = xmlhttp.response.querySelector('meta[http-equiv="refresh"]');
```

Comment:
`document.querySelector('meta[http-equiv="refresh"]')` in Chrome for uppercase attribute seems to work without a problem, but I still added `i` just in case.